### PR TITLE
Implement /load without params to show some info

### PR DESCRIPTION
**Pull request by def-**
about your existing saves. Not the code of course, but only how many saves you have and when you last saved on this map.

### CODE REVIEW

**Comment by heinrich5991** [comment]
IMO better suited for /load. Contains obvious thread-unsafety, but I think we accept that until someone rewrites the SQL stuff.

Security impact. Allows to gain information previously not covered. Might make it easier to brute-force save codes. I don't see a direct way though, so probably(?) fine.

**Comment by def-** [comment]
> IMO better suited for /load.

Done

**Comment by Learath2** [comment]
LGTM

bors r+

File: src/game/server/score/sql_score.cpp [hunk 1]
```diff
@@ -1635,4 +1635,60 @@ bool CSqlScore::LoadTeamThread(CSqlServer* pSqlServer, const CSqlData *pGameData
...
+	Tmp->m_Name = Server()->ClientName(ClientID);
+
+	thread_init_and_detach(ExecSqlFunc, new CSqlExecData(GetSavesThread, Tmp, false), "get saves");
+}
+
+bool CSqlScore::GetSavesThread(CSqlServer* pSqlServer, const CSqlData *pGameData, bool HandleFailure)
+{
+	const CSqlGetSavesData *pData = dynamic_cast<const CSqlGetSavesData *>(pGameData);
+
+	if (HandleFailure)
```
Comment (Learath2) [inline src/game/server/score/sql_score.cpp:1651]:
```suggestion
	if(HandleFailure)
```

Comment (def-) [reply src/game/server/score/sql_score.cpp:1651]: Then I have to go through sql_score.cpp and change it all. This is the same as in every other function there.

Comment (Learath2) [reply src/game/server/score/sql_score.cpp:1651]: Hm, I guess not worth the trouble, we can maybe fix it all if we decide to adopt a `.clang-format` sometime.

**Pull request merged**